build.gradle.kts

```diff
@@ -4,14 +4,17 @@
  */
 import io.gitlab.arturbosch.detekt.Detekt
 
+buildscript {
+    val compose_version by extra("1.1.0-beta01")
```

Comment:
⚠️ **[detekt]** <detekt.VariableNaming> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Variable names should match the pattern: [a-z][A-Za-z0-9]*